.clang-format

```diff
@@ -0,0 +1,274 @@
+---
+Language:        Cpp
```

Comment:
Should we use C here? Does it matter?

Reply:
I don't think it matters much - I think the defaults are pretty sane for C and C++.